main.js

```diff
@@ -455,7 +455,7 @@
     function fetchAllData() {
         Promise.all([
             fetchWPRDCData("Police", { limit: 250 }),
-            fetchWPRDCData("311", { limit: 250 }),
+            fetchWPRDCData("311"),
```

Comment:
If the updated ".primaryFilter" that I used for the 311 data-set works for the others, we shouldn't need this limit for any of them. I'll test it out and get back to you!